tutorials/pages/create-nft-with-iota-sdk.md

```diff
@@ -0,0 +1,220 @@
+
+:::note
+
+We broke the code into separate snippets to help you understand it better. To make it work, copy all code snippets one after another into the file that you have just created.
+
+:::
+
+#### 1. Imports and parameters
+
+```javascript
+// Libraries
+import {
+  AddressUnlockCondition,
+  Ed25519Address,
+  IssuerFeature,
+  MintNftParams,
+  SenderFeature,
+  utf8ToHex,
+  Utils,
+  Wallet,
+} from '@iota/sdk';
+const IPFS = require('ipfs-core');
+const fs = require('fs');
+
+// Environment variables
+require('dotenv').config({ path: './.env' });
+```
+
+#### 2. Upload file to IPFS
+
+The function `uploadByPath()` starts a local IPFS node, reads the file from the provided path, uploads it to IPFS and returns the unique IPFS content identifer.
+
+```javascript
+async function uploadByPath(filePath) {
+  try {
+    console.log('\n');
+    console.log(consoleColor, `Start local IPFS node for upload:`);
+
+    // Set up local IPFS node for upload
+    let node;
+    if (!node) {
+      node = await IPFS.create({
+        repo: `ipfs_node`,
+      });
+    }
+
+    // Read file from path
+    const file = fs.readFileSync(filePath);
+
+    // Upload file to IPFS
+    const fileAdded = await node.add(file);
+    const contentIdentifier = fileAdded.path;
+
+    console.log('\n');
+    console.log(
+      consoleColor,
+      `Your file was uploaded to IPFS with the following Content Identifier (CID):`,
+    );
+    console.log(contentIdentifier, '\n');
+
+    console.log(consoleColor, `Check your file on IPFS:`);
+    console.log(`https://ipfs.io/ipfs/${contentIdentifier}`, '\n');
+
+    return contentIdentifier;
+  } catch (error) {
+    console.error('IPFS upload error', error);
+  }
+}
+```
+
+#### 3. Prepare NFT metadata
+
+This part calls the `uploadByPath()` function described above and prepares the metadata of your NFT. Make sure to give your NFT a nice name here.
+
+```javascript
+async function run() {
+    try {
+        const filePath = "nft-image.jpg";
+        const ipfsCid = await uploadByPath(filePath);
+
+        // Define NFT metadata
+        const metadataObject = {
+            standard: "IRC27",
+            type: "image/jpeg",
+            version: "v1.0",
+            name: "<Enter_your_desired_name_here>",
+            uri: `https://ipfs.io/ipfs/${ipfsCid}`
+        };
+
+        const metadataBytes = utf8ToHex(JSON.stringify(metadataObject));
+```
+
+#### 4. Import Stronghold account and synchronize
+
+```javascript
+if (!process.env.STRONGHOLD_PASSWORD) {
+  throw new Error('.env STRONGHOLD_PASSWORD is undefined, see .env.example');
+}
+
+const wallet = new Wallet({
+  storagePath: process.env.WALLET_DB_PATH,
+});
+
+const account = await wallet.getAccount('Alice');
+
+// We send from the first address in the account.
+const senderAddress = (await account.addresses())[0].address;
+
+// We need to unlock stronghold.
+await wallet.setStrongholdPassword(process.env.STRONGHOLD_PASSWORD);
+```
+
+#### 5. Mint NFT
+
```

Comment:
This could do with an explanation. Why  do you use `MintNftParams`?  Where do the values you set come from? Why do you have to prepare and then send the transaction?